UI/AppKit/Interface/LadybirdWebView.mm

```diff
@@ -954,6 +954,11 @@ - (void)setWebViewCallbacks
         [NSMenu popUpContextMenu:self.select_dropdown withEvent:event forView:self];
     };
 
+    m_web_view_bridge->on_set_browser_zoom = [](double factor) {
+        (void)factor;
+        dbgln("FIXME: A test called `window.internals.setBrowserZoom()` which is not implemented in the AppKit UI");
```

Comment:
Ideally this FIXME would get implemented as part of this PR, I thunk. But I don't have access to a Mac, so I can't do it myself. And I'd like to get some feedback on the general approach first, before trying to find someone who can implement the Mac version.